### PR TITLE
Feature add user AB#93

### PR DESCRIPTION
**Pull request by markusranda**
- Refactored adminpage, which now implements a grid, instead of list.
- Add button which opens a form for registrating new users.
- Successfully sends authorized request to server.
- Feedback to user for registrating new users.

Closes [AB#93](https://dev.azure.com/ReleaseNoteSystem/399f705f-cd58-45f2-becb-f890cb50f774/_workitems/edit/93)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,14 +24,14 @@ Axios.defaults.baseURL = "http://localhost:5000/api/";
 
 // intercept outgoing and incoming requests for debugging
 Axios.interceptors.request.use(request => {
-  console.log('Starting Request', request)
-  return request
-})
+  console.log("Starting Request", request);
```
Comment (Mordi490) [inline src/App.js]: bør dette være i development branchen?

Comment (markusranda) [reply src/App.js]: Kan egentlig se det samme ved å gå inn i tabben "Network", så treng den faktisk ikke tell debugging engang. Eg fjern den.

Comment (markusranda) [reply src/App.js]: Eller, kerru sei @0us ?

File: src/components/adminpanel/AddUserForm.jsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+const TurboPaper = styled(Paper)`
+  padding: 1rem 3rem 1rem 3rem;
+  @media only screen and (max-width: 600px) {
+    padding: 0;
+  }
+`;
+
+const TurboButton = styled(Button)`
+  && {
```
Comment (0us) [inline src/components/adminpanel/AddUserForm.jsx]: useriøst og barnslig x---D

File: src/components/screen/AdminScreen.jsx [hunk 3]
```diff
@@ -1,75 +1,109 @@
...
+        this.createData("Trond Viggo Torgersen"),
+        this.createData("Sinnasnekkern")
+      ]
+    };
+  }
 
-    return table;
+  createData = (name, code, population, size) => {
+    const density = population / size;
+    return { name, code, population, size, density };
```
Comment (0us) [inline src/components/screen/AdminScreen.jsx]: Ka e ditta her

Comment (markusranda) [reply src/components/screen/AdminScreen.jsx]: Ditta e gammal kode. Kan fjerna de

**Pull request merged**